app/elements/common/data-providers/team/teams-for-user-provider.html

```diff
@@ -25,10 +25,13 @@
 				Behaviors.Providers.General
 			],
 			observers: [
-				'_userIdChanged(userId)'
+				'_usernameChanged(username)'
 			],
-			_userIdChanged: function(userId){
-				this.url = this.baseUrl + '/users/' + userId + '/teams';
+			_usernameChanged: function(username){
```

Comment:
no, but it changes when it receives the baseUrl and then when it receive the username